src/middlewared/middlewared/plugins/crypto_/certificates.py

```diff
@@ -595,7 +595,7 @@ async def do_update(self, job, id, data):
 
         new.update(data)
 
-        if any(new[k] != old[k] for k in ('name', 'revoked')):
+        if any(new.get(k) != old.get(k) for k in ('name', 'revoked', 'renew_days')):
```

Comment:
What if both new.get(k) and old.get(k) are None?

Reply:
It means nothing changed, we are good